resources/qml/SidebarHeader.qml

```diff
@@ -243,6 +244,13 @@ Column
             }
         }
 
+        Text  // to take the space of the material info button when the active machine doesn't have multiple extruders
```

Comment:
A single extrusion machine should also show it guess. It can have materials right?